config.ru

```diff
@@ -1,6 +1,8 @@
+require 'newrelic_rpm'
 require 'cellect/server'
 require_relative 'lib/cellect/server/adapters/panoptes'
 
+p ENV['RAILS_ENV']
```

Comment:
Not sure this ENV var is being set but if we want to print it out we can move it to here https://github.com/zooniverse/cellect_panoptes/blob/e9012517fd71d001ac6e3c1621ee9e8af50b4056/start#L13

Which is triggered by the DEBUG_CELLECT_START env variable.